--- origsrc/PAR-Packer-1.034/myldr/embed_files.pl	2016-07-16 10:46:50.000000000 -0400
+++ src/PAR-Packer-1.034/myldr/embed_files.pl	2016-09-25 16:43:02.141406600 -0400
@@ -40,7 +40,7 @@ for ($^O)
         {
             chomp(my $system_root = qx( cygpath --unix '$ENV{SYSTEMROOT}' ));
             print STDERR "### SystemRoot (as Unix path) = $system_root\n";
-            *is_system_lib = sub { shift =~ m{^/usr/bin/|^\Q$system_root\E/}i };
+	    *is_system_lib = sub { shift =~ m{^/usr/bin/(?!cygcrypt\b)|^\Q$system_root\E/}i };
         }
         else
         {
